dotnet-desktop-guide/framework/winforms/controls/how-to-create-an-unbound-windows-forms-datagridview-control.md

```diff
@@ -26,9 +24,9 @@ The following code example demonstrates how to populate a <xref:System.Windows.F
   
 ## Compiling the Code
 
- This example requires:
+This example requires:  
   
-- References to the System, System.Drawing, and System.Windows.Forms assemblies.
+References to the System, System.Drawing, and System.Windows.Forms assemblies.  
```

Comment:
Please remove this space you added. We need to stick to the scope outlined in the user story.

Reply:
removed the empty space here.

Reply:
Sure.